payment/internal/repository/repository_user.go

```diff
@@ -0,0 +1,22 @@
+package repository
+
+import (
+	"github.com/GrabItYourself/giys-backend/lib/postgres/models"
+)
+
+func (r *Repository) GetUserById(id string) (*models.User, error) {
```

Comment:
ok, we have to start think whether or not to move repository codes to lib, or should we invoke UserService to `GetUserById` (instead of through postgres directly from other services)
cc @saenyakorn @natTP

Reply:
Moving repository code to lib is simpler, and every service can just use the same repo code, but it may defeat the meaning behind using microservice (where every service handles one thing)

Reply:
i like just moving repo code to lib and share though, it's easier. But idk if it'll pose a problem with the professors

Reply:
I think sharing repo code between microservices doesn’t defeat the meaning behind using microservice architecture yet, since our design still separates the domain responsibilities - each service is responsible for its own domain. The concept is still correct I think

Reply:
will there be a problem if 2 services can write the same table?

Reply:
> will there be a problem if 2 services can write the same table?

maybe not i think